variables.tf

```diff
@@ -168,6 +168,12 @@ variable "log_events" {
   default     = false
 }
 
+variable "log_level" {
```

Comment:
this variable doesn't appear to be connected to the function as described - if a user changes this to `INFO`, how does that get propagated?